src/main/kotlin/nl/info/zac/search/model/FilterVeld.kt

```diff
@@ -11,6 +11,7 @@ enum class FilterVeld(val veld: String) {
     GROEP("groepNaam"),
     TOEGEKEND("isToegekend"),
 
+    ZAAK_IDENTIFICATIE("zaak_identificatie"),
```

Comment:
This field does not seem to be used in this PR at all? What's the deal here?

Reply:
Used in method  nl.info.zac.app.zaak.ZaakKoppelenRestService.buildZoekParameters